<launch>
  <node pkg="joy" type="joy_node" name="joy_node" output="screen">
  </node>
  <node pkg="realman_sample" type="rm_teleop" name="rm_teleop" output="screen">
    <param name="linear_scale"   value="0.05"/>
    <param name="deadzone"   value="0.05"/>
  </node>
</launch>
